Lesson2/hw2.py

```diff
@@ -31,6 +31,3 @@ def transform(legacy_data: Dict[int, List[str]]) -> Dict[str, int]:
             output.update(dict.fromkeys(value.lower(), key))
 
     return dict(sorted(output.items()))
```

Comment:
No need to sort the dict, you can just return `output` right away. Dicts are compared without order, so tests will pass

Reply:
Got it, thanks!